packages/gravity-ui-nunjucks/src/_patterns/03-organisms/00-page-structure/00-page-header.njk

```diff
@@ -21,7 +21,7 @@
         {%- endif %}
       </div>
       {{ toggleMenu(navMenuExpanded) }}
-      {{ navMenu(links, externalLinks) | safe }}
+      {{ navMenu(links) | safe }}
```

Comment:
I'd rever this for the same reason as above.

Our site (and the page examples, if you update their JSON) won't use external links, but other Gravity-powered sites might, so we should retain that option.